examples/fedpca_examples/perform_pca/README.md

```diff
@@ -0,0 +1,41 @@
+# Federated Principal Component Analysis Example
+This example performs federated principal component analysis. The goal is to compute the principal components of a subset of the MNIST dataset, under the condition that the data is distributed across two distinct clients.
```

Comment:
It looks like in the config.yaml we actually have four clients taking part in the FedPCA calculations? Perhaps I am misinterpreting the comment though.

Reply:
Ah yes. Originally both of the FedPCA examples involve only two clients. After I locally tested for more clients, I committed those changes but forgot to update the README. I'll update them so they are consistent with the config file. Sorry for the confusion 😂